c/tskit/tables.c

```diff
@@ -5035,21 +5054,55 @@ tsk_table_sorter_sort_individuals(tsk_table_sorter_t *self)
         }
     }
 
+out:
+    tsk_safe_free(incoming_edge_count);
+    return ret;
+}
+
+static int
+tsk_table_sorter_sort_individuals(tsk_table_sorter_t *self)
+{
+    int ret = 0;
+    tsk_id_t i;
+    tsk_individual_table_t copy;
+    tsk_individual_t individual;
+    tsk_individual_table_t *individuals = &self->tables->individuals;
+    tsk_node_table_t *nodes = &self->tables->nodes;
+    tsk_size_t num_individuals = individuals->num_rows;
+    tsk_id_t *traversal_order = malloc(num_individuals * sizeof(*traversal_order));
+    tsk_id_t *new_id_map = malloc(num_individuals * sizeof(*new_id_map));
+
+    if (new_id_map == NULL || traversal_order == NULL) {
+        ret = TSK_ERR_NO_MEMORY;
+        goto out;
+    }
+    memset(new_id_map, 0xff, num_individuals * sizeof(tsk_id_t));
```

Comment:
Might as well use ``sizeof(*new_id_map))`` here for consistency with mallocs above